masonry_core/src/core/action.rs

```diff
@@ -28,7 +28,7 @@ pub enum Action {
     CheckboxToggled(bool),
     // FIXME - This is a huge hack
     /// Other.
-    Other(Box<dyn Any + Send>),
+    Other(Box<dyn AnyDebug + Send>),
```

Comment:
Oh. I thought the `Action` enum itself might go away.

Reply:
That's a bigger change, which this is just one part of. But yes, the current plan is for that enum to go away.

My planned next step here is to automatically unwrap the `Other` variant inside Xilem.